test-utils/src/main/kotlin/com/google/devtools/ksp/processor/AnnotationTargetsProcessor.kt

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2020 Google LLC
+ * Copyright 2010-2020 JetBrains s.r.o. and Kotlin Programming Language contributors.
```

Comment:
Just to double-check, do these intentionally stay in 2020, or is it a good time to update the template somewhere to make this copyright comment up-to-date?